pkg/placement/tables.go

```diff
@@ -27,7 +29,7 @@ type HostInfo struct {
 }
 
 // GetPlacementTables returns the current placement host infos.
-func (p *Service) GetPlacementTables() (*PlacementTables, error) {
+func (p *Service) GetPlacementTables() ([]byte, error) {
```

Comment:
What's the purpose of this change in relation to the PR?

Reply:
This is because the healthz server extra endpoints option has been generalized to now just accept a `([]byte, error)` "`Getter`". I think this is cleaner/easier to consume. This function was only ever used for that anyway, so makes sense to json marshal in-func also.